### DISCUSSION TITLE
Remove the Like/Reblog/etc buttons hover text

### DISCUSSION THREAD

**Discussion by faefangirl** [AprilSylph/XKit-Rewritten#575] [Suggestion Box]
I would like to see a feature that turns off the hover animation/text when you hover over the Like button. The hover delay often causes my likes to not register and I have to like again or wait for the animation.

**Comment by AprilSylph** [comment]
~~I was going to implement this but it looks like the footer redesign part 2 is rolling out already, which removes these anyway. Not everyone has this yet, but I don't imagine it's going to be unpopular enough to be rolled back (especially given the public consensus on the first redesign).~~

Nevermind, they're still there.
